oam/install_scripts/columnstore

```diff
@@ -56,68 +56,68 @@ if [ "x$has_um" = x ]; then
 	has_um=0
 fi
 
-user=`whoami 2>/dev/null`
-SUDO=" "
-if [ $user != "root" ]; then
-        SUDO="sudo "
-fi
+lockdir=`$InstallDir/bin/getConfig Installation LockFileDirectory`
+
+#get temp directory
+tmpDir=`$InstallDir/bin/getConfig SystemConfig SystemTempFileDir`
+mkdir $tmpDir >/dev/null 2>&1
```

Comment:
Not that important, but we can just use the tmp dir as the lock dir.  I can't think of a reason to make them separate.  Simpler is better.